specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/networkWatcher.json

```diff
@@ -3192,6 +3192,10 @@
           "description": "ID of the storage account which is used to store the flow log.",
           "type": "string"
         },
+        "enabledFilteringCriteria": {
+          "description": "Optional condition to filter flowlogs",
```

Comment:
You should have a fuller description that documents the syntax of this string, or links to related docs, otherwise people won't know how to use it.

Reply:
we will publish the documentation about this when the feature is out to the customers for use.

Reply:
Hmm, how will you get your spec and SDKs, all updated to point to it? Would an aka.ms placeholder link be a good aid in getting that done smoothly?

Reply:
I am not sure I understand this comment.
Are you asking us to add link to the documentation in description of the field?